internal/host/statemachine.go

```diff
@@ -196,6 +196,14 @@ func NewHostStateMachine(th *transitionHandler) stateswitch.StateMachine {
 		PostTransition:   th.PostResettingPendingUserAction,
 	})
 
+	sm.AddTransition(stateswitch.TransitionRule{
```

Comment:
it should return an error. you should not try to disable disabled host

Reply:
@avishayt i don't remember if it was you or someone else that required that there will be no dummy transition for disable disabling host.
That if user try to disable already disabled host it will fail